### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by romanofski**
A few patch which clean up deprecated files and update documentation.

### CODE REVIEW

**Review by sthaha — COMMENTED** [review]
Do you know if we can use travis or any other fedora service to build the rpm?

**Comment by romanofski** [comment]
Currently working on a way to get travis to build the rpms using https://hub.docker.com/r/mmornati/mock-rpmbuilder/. Not sure if travis supports artefacts, but at least I will know if the rpm finished building successfully or not.

File: README.asciidoc [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
-===================
+RPMGrill
+========
 :doctype: book
 :toc:
 :lang: en
 
+Overview
+--------
+
```
Comment (sthaha) [inline README.asciidoc:9]: How about adding the travis link: https://travis-ci.org/default-to-open/rpmgrill.svg?branch=master

**Comment by romanofski** [comment]
Had to make two changes:

1) I forgot to include the README.AAA_FIRST change to README.asciidoc in the perl build artifacts and
2) The rpmgrill.spec had the release encoded in the %Source macro, which was due to my mistake when I released and packaged 0.28. The release shouldn't be part of the source release.

File: README.asciidoc (hunk 1 above)
Comment (romanofski) [reply README.asciidoc:9]: You sure? Always felt those tags were a bit corny, especially once the builds stop passing ;)

Comment (sthaha) [reply README.asciidoc:9]: I like those because it is good indicator to a potential contributor that there are tests and are run continuously

Comment (romanofski) [reply README.asciidoc:9]: Alright, fair enough. Here we go.

**Review by sthaha — APPROVED** [review]

**Pull request merged**